packages/core/src/extensions/UniqueID/UniqueID.ts

```diff
@@ -75,6 +75,12 @@ const UniqueID = Extension.create({
         attributes: {
           [this.options.attributeName]: {
             default: null,
+            parseHTML: (element) =>
```

Comment:
why is this added here?

Reply:
Without it, we have to use `BlockAttributes` to translate node attributes to HTML attributes, which can mess with `blockContainer` attributes and should really just be done using the `parseHTML` & `renderHTML` functions. Adding those to `UniqueID` greatly simplifies the `renderHTML` function of `blockContainer` and makes it less prone to causing bugs if new attributes are added in the future.